spec/image_spec.rb

```diff
@@ -46,6 +46,23 @@
     expect(@jp2.width).to eq 100
   end
 
+  it 'should create jp2 with a temp file when given a compressed RGB tif' do
```

Comment:
Avoid using `should` in rspec descriptions
 <img src="https://media.giphy.com/media/pvDp7Ewpzt0o8/giphy.gif">